src/js/components/Tabs/__tests__/__snapshots__/Tabs-test.js.snap

```diff
@@ -2615,6 +2615,7 @@ exports[`Tabs should apply custom theme disabled style when theme.button.default
         aria-expanded="true"
         aria-selected="true"
         class="c3"
+        kind="default"
```

Comment:
How are these changes related to this PR?

Reply:
Unrelated. This has been resolved.